assert/assertions_test.go

```diff
@@ -2687,7 +2793,7 @@ func TestFailNowWithFullTestingT(t *testing.T) {
 }
 
 func TestBytesEqual(t *testing.T) {
-	var cases = []struct {
+	cases := []struct {
```

Comment:
I'm pretty sure you are using gofumpt as I do 😁

Reply:
hehe, indeed

Reply:
also here: https://github.com/stretchr/testify/pull/1690/commits/bdff01450338f98f2c043d0c08dda4de1ddc9fda